### PR TITLE
package mariadb-10.3: rebuild against the latest MariaDB 10.3 package

### PR DESCRIPTION
**Pull request by komainu8**
We need to increase minor version of Debian package for Mroonga when we update Debian package for Mroonga.

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -138,8 +138,16 @@ jobs:
...
+            debian-*)
+              deb_release=$(head -n1 packages/${{ matrix.package }}-mroonga/debian/changelog | \
+                            awk '{print $2}' | \
+                            awk -F "-" '{print $2}' | \
+                            sed -e 's/[^0-9]//g')
+              ;;
+          esac
           cd packages/${{ matrix.package }}-mroonga
-          rake ${{ matrix.package-type }}:build BUILD_DIR=build
+          rake ${{ matrix.package-type }}:build BUILD_DIR=build DEB_RELEASE=${deb_release}
```
Comment (kou) [inline .github/workflows/linux.yml:150]: Why do we need to specify `DEB_RELEASE` here?
`DEB_RELEASE` is for `rake version:update` not `rake apt:build`.

**Comment by kou** [comment]
CI is failed because of the target branch isn't master.
This will work on master and `*-maintainance` branch.

File: .github/workflows/linux.yml [hunk 2]
```diff
@@ -119,6 +119,8 @@ jobs:
         run: |
           make dist
       - name: Update version
+        if: |
+          github.ref == 'refs/heads/master'
```
Comment (kou) [inline .github/workflows/linux.yml:123]: Renaming maintenance branch name rule to `maintenance/11.03` from `v11.03-maintenance` and changing this to `startsWith(github.ref, 'refs/tags/') || startsWith(github.ref, 'refs/heads/maintenance/')` may be better.

Comment (komainu8) [reply .github/workflows/linux.yml:123]: "Update version" execute only on master.
So, I think that the following code is correct. Is that my misunderstanding?

```
if: |
  !startsWith(github.ref, 'refs/tags/') && !startsWith(github.ref, 'refs/heads/maintenance/')
```

Comment (kou) [reply .github/workflows/linux.yml:123]: Right.
My code was wrong.

**Pull request merged**